docs/keystatic.config.tsx

```diff
@@ -82,6 +82,11 @@ export const componentBlocks = {
         label: 'Sizes',
         description: 'Optionally override the defualt sizes',
```

Comment:
This is not from your changes but I saw it so have to fix it 😂 

```suggestion
        description: 'Optionally override the default sizes',
```